modules/ROOT/pages/configure-notifications.adoc

```diff
@@ -13,13 +13,16 @@ Each email notification includes:
 
 Enable and configure the notifications to automatically send email when the latest version of an API becomes non-conformant. 
 
-NOTE: Notifications are sent only if there is a valid contact email address configured for the selected recipient.
+API Governance sends notifications about conformance only when:
 
-Automated notifications are sent for each ruleset that has a new conformance failure when the conformance status changes during validation. A notification is sent for a specific API only the first time that version of the API fails conformance to a specific ruleset. 
+* One or more active profiles targets the API.
+* There is a valid contact email address configured for the selected recipient.
+* An API's conformance status changes to fail because of a change in the API or applied rulesets.
+* The first time a specific version of an API fails conformance to a specific ruleset. 
 
 Following are the changes that might result in notifications: 
 
-. Updating a governance profile, including adding or removing rulesets 
+. Updating an active governance profile, including adding or removing rulesets 
```

Comment:
Should these list items be bullets rather than numbers?

Consider using imperative and terminating punctuation.

Reply:
Yes on the first.
Since these might happen by other people in different ways, I think the current wording and construct makes the most sense.